docs/en/getting-started/get-started-stack.asciidoc

```diff
@@ -200,16 +212,22 @@ To learn more about installing, configuring, and running {es}, read the
 https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html[{es} Reference].
 
 [float]
-==== Make sure {es} is up and running
-
+==== Make sure that {es} is up and running
 
-To test that the {es} daemon is up and running, try sending an HTTP GET
-request on port 9200.
+Open a new terminal and verify that you can connect to your {es} cluster by
+making an authenticated call. Enter the password for the `elastic` user when
+prompted:
 
 [source,shell]
-----------------------------------------------------------------------
-curl http://127.0.0.1:9200
-----------------------------------------------------------------------
+----
+curl --cacert config/tls_auto_config_<timestamp>/http_ca.crt \
```

Comment:
we haven't told them that they need to be in the dir where they installed elasticsearch so a relative path `config/tls_auto_config_<timestamp>/http_ca.crt` is confusing. We should either tell them to navigate to that dir or (preferably IMO) qualify the path with $ES_CONF_PATH (linking to where we describe where that points to in each installation type.

Reply:
++ I think we can change this to:

>Path to the generated `http_ca.crt` certificate for the HTTP layer. This file is created in the {es} configuration directory, which is defined by the`$ES_PATH_CONF` [environment variable](https://www.elastic.co/guide/en/elasticsearch/reference/8.0/settings.html#config-files-location).